locale/en.po

```diff
@@ -5297,6 +5297,9 @@ msgstr "Silica"
 msgid "SILICA_MEMBRANE_DESCRIPTION"
 msgstr "This membrane has a strong wall of silica. It can resist overall damage well and is very resistant to physical damage. It also requires less energy to maintain its form. However, it slows the cell down by a large factor and the cell absorbs resources at a reduced rate."
 
+msgid "SIMULATION"
```

Comment:
I think this needs a *slightly* more descriptive name, perhaps "SIMULATION_CONFIG" might be good? Or "SIMULATION_TITLE"